juriscraper/opinions/united_states/state/pasuperct.py

```diff
@@ -39,3 +41,23 @@ def get_status(self, op: Dict) -> str:
         if descr == "Precedential":
             return "Published"
         return "Unknown"
+
+    def clean_judge(self, author_str: str) -> str:
+        """
+        Examples:
+        Input: "Wallace, J."
+        Output: "Wallace"
+
+        Input: "Leadbetter, President Judge Emerita"
+        Output: "Leadbetter"
+
+        Input: "Cohn Jubelirer, President Judge ~ Concurring and Dissenting Opinion by McCullough, J."
+        Output: "Cohn Jubelirer; McCullough"
+        """
+        regex = r"(?P<judge>\w+\s*\w*),\s+(President|(P\.)?J\.)"
+        match = re.finditer(regex, author_str)
+        if match:
+            return ". ".join([m.group("judge") for m in match]).replace(
+                " by ", " "
+            )
+        return author_str
```

Comment:
this feels like something that should be in the judge parsers section of jurisraper

Reply:
I tried the judge parsers function and they don't work
```
In [1]: inputs = ["Wallace, J.", "Leadbetter, President Judge Emerita", "Cohn Jubelirer, President Judge ~ Concurring and Dissenting Opinion by McCullough, J."]

In [2]: from juriscraper.lib.judge_parsers import normalize_judge_string, normalize_judge_names

In [3]: for i in inputs: print(normalize_judge_names(i))
Wallace, J.
Leadbetter, President Judge Emerita
Cohn Jubelirer, President Judge ~ Concurring and Dissenting Opinion by McCullough, J.

In [4]: for i in inputs: print(normalize_judge_string(i))
('Wallace J.', 'jud')
('Leadbetter President Emerita', 'jud')
('Cohn Jubelirer President Concurring and Dissenting Opinion by McCullough J.', 'jud')
```

Also, I am taking advantage of specific things from these sources, that I don't think would generalize to other sources
- that judge last names are followed by their title, like ", J." or ",P.J." or "President"
- that at most, there are 2 words for the judge name. If I caught a " by ", I erase it